src/components/reusable/NavBar.js

```diff
@@ -0,0 +1,40 @@
+import { React } from "react";
+import aqLogo from "../../assets/images/aq-logo.png";
+import { Link } from "react-router-dom";
+import { useAuthenticator } from "@aws-amplify/ui-react";
+
+const Navbar = () => {
+  const { signOut } = useAuthenticator((context) => [context.user]);
+
+  return (
+    <nav className="fixed top-0 w-full z-50 px-4 py-2 bg-dark-gray border-gray-300 shadow">
+      <div className="container flex flex-wrap justify-between items-center mx-auto">
+        <a href="http://localhost:3000/" className="flex items-center">
```

Comment:
This should actually link to the home page or wherever you wanted it to go now that we have routing setup.